packages/portal/src/components/search/SideFilters.vue

```diff
@@ -511,6 +524,9 @@
       toggleFilterSheet() {
         this.$store.commit('search/setShowFiltersSheet', !this.$store.state.search.showFiltersSheet);
       },
+      toggleAdvancedSearch() {
+        this.$store.commit('search/setShowAdvancedSearch', !this.$store.state.search.showAdvancedSearch);
```

Comment:
Let's not use the store for this, but instead emit an event for SearchInterface to react to.

Alternatively, even that may not be needed if the toggle button is moved into the slot.

Reply:
Toggle button moved into the slot and state handled in `SearchInterface`.